### PR TITLE
Update Docker examples to use Compose v2 command

### PR DESCRIPTION
**Pull request by maxproske**
Compose v1 has reached EOL, and will no longer be included with Docker after June 2023: https://www.docker.com/blog/docker-desktop-4-19/

### CODE REVIEW

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 8c112eff29083a44e9a058a18728594d0a7e2225

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 8c112eff29083a44e9a058a18728594d0a7e2225

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

File: examples/with-docker-compose/README.md [hunk 1]
```diff
@@ -46,12 +46,10 @@ First, run the development server:
...
 
 # Build dev
-# Note: Keep v1 command until "Use Docker Compose v2" is enabled by default for Docker Desktop for Linux
-# Docker aliases `docker-compose` (v1 command) to `docker compose` (v2 command), but not the other way around
-docker-compose -f docker-compose.dev.yml build
+docker compose -f docker-compose.dev.yml build
 
 # Up dev
-docker-compose -f docker-compose.dev.yml up
+docker compose -f docker-compose.dev.yml up
```
Comment (Hasnain01-hub) [inline examples/with-docker-compose/README.md:52]: /review

File: examples/with-docker-compose/README.md [hunk 2]
```diff
@@ -70,10 +68,10 @@ First, run the production server (Final image approximately 110 MB).
 docker network create my_network
 
 # Build prod
-docker-compose -f docker-compose.prod.yml build
+docker compose -f docker-compose.prod.yml build
 
 # Up prod in detached mode
-docker-compose -f docker-compose.prod.yml up -d
+docker compose -f docker-compose.prod.yml up -d
 ```
```
Comment (Hasnain01-hub) [inline examples/with-docker-compose/README.md:75]: /review

**Review by ijjk — APPROVED** [review]
Thanks!

**Pull request merged**